### PR TITLE
 Alert-Window-Implementation

### PR DESCRIPTION
**Pull request by thebkaviya**

### CODE REVIEW

File: hydrator.py [hunk 1]
```diff
@@ -1,19 +1,12 @@
+
+from curses import window
```
Comment (tarithj) [inline hydrator.py:2]: Looks like a unused import

**Comment by tarithj** [comment]
You should prob add tkinter to the Pipfile

File: hydrator.py (hunk 1 above)
Comment (Binula-Kavisinghe) [reply hydrator.py:2]: oh yeah, i forgot to remove that..

**Comment by Binula-Kavisinghe** [comment]
> You should prob add tkinter to the Pipfile

yeah, I'll get back to it after the exams...

**Comment by thebkaviya** [comment]
@tarithj please update the dependencies too...

File: hydrator.py [hunk 2]
```diff
@@ -1,30 +1,44 @@
...
     This program comes with ABSOLUTELY NO WARRANTY.
     This is free software, and you are welcome to redistribute it
     under certain conditions;
     
     This software uses "Notification" by IENB ( https://freesound.org/people/IENBA/sounds/545495/ ) from freesound licensed under CCBY3.0
 
-    You will hear a tone in every hour
-""")
-play_notification()
+    You will hear a tone and see a popup in every hour.
```
Comment (tarithj) [inline hydrator.py:22]: remove "in", "see a popup in every hour" sounds wrong maybe "see a popup every hour?"

Comment (thebkaviya) [reply hydrator.py:22]: noted

File: Pipfile [hunk 3]
```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 playsound = "*"
+PySimpleGUI = "*"
```
[the remark this answers is not in the document]
Comment (thebkaviya) [reply Pipfile:8]: aand how would you do that ?

**Comment by thebkaviya** [comment]
> @tarithj please update the dependencies too...

...

**Review by tarithj — APPROVED** [review]

**Pull request merged**